doc/reference/runtime_conf/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _runtime_config_reference:
+
+
+Runtime Configuration System
+############################
+
+Zephyr-RTOS provide common submodule called 'settings' for store persistently
```

Comment:
Fixing some misspellings and grammar:
```
Zephyr RTOS provides a submodule called 'settings' used
persistent storage such as saving and loading runtime
configuration data. A variety of storage implementations
are provided behind a common API using FCB, NVS, or a
file system.  These different implementations give the
application developer flexibility to select an appropriate
storage medium, and even change it later as needs change.

The runtime configuration system is used by various Zephyr
components and can be used simultaneously by user applications.
```

Did I get the details right?

Reply:
^^ looks fine to me

Reply:
OK, if you update your PR with this change, the doc is ready...

Reply:
@dbkinder update